src/datetime/serde.rs

```diff
@@ -25,7 +26,9 @@ pub struct MicroSecondsTimestampVisitor;
 #[derive(Debug)]
 pub struct MilliSecondsTimestampVisitor;
 
-/// Serialize into a rfc3339 time string
+/// Serialize into a rfc3339 time string.
+/// In addition to rfc3339 the offset may contain seconds when present, instead of only hours and
+/// minutes.
```

Comment:
This documents the current behavior, that I had to re-implement to make a test pass.

But deserialization is unable to read on offset with seconds. What should we do here?
Being able to round-trip is worth something, writing correct rfc3339 time strings is also good.

Reply:
I opened https://github.com/chronotope/chrono/pull/1036 with a proposal to deny offsets with seconds, which should never exist anyway.